src/main/java/com/github/manolo8/darkbot/config/ImageWrapper.java

```diff
@@ -0,0 +1,33 @@
+package com.github.manolo8.darkbot.config;
+
+import java.awt.Image;
+import javax.swing.ImageIcon;
+
+public class ImageWrapper {
+    private String path;
+    private transient Image image;
+    private transient boolean loaded;
+
+    public String getPath() {
+        if (this.path == null)
+            return "None";
+        return this.path;
+    }
+
+    public void setPath(String path) {
+        this.path = path;
+        this.loaded = false;
+    }
```

Comment:
i believe gson will serialize it by using these 2 methods, which means an empty image will serialize as the string "None"

I think getPath should return null if it is indeed null (handle displaying a "None" on the editor, not here)